Spigot-Server-Patches/0418-Implement-PlayerTradeWithVillagerEvent.patch

```diff
@@ -0,0 +1,29 @@
+From e09883ac8ac9d4bd2a2a1289ec447748ed09148c Mon Sep 17 00:00:00 2001
+From: bermudalocket <bermudalckt@gmail.com>
+Date: Sun, 17 Feb 2019 14:57:27 -0500
+Subject: [PATCH] Implement PlayerTradeWithVillagerEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityVillager.java b/src/main/java/net/minecraft/server/EntityVillager.java
+index 40b3ffd8..6f6ab55a 100644
+--- a/src/main/java/net/minecraft/server/EntityVillager.java
++++ b/src/main/java/net/minecraft/server/EntityVillager.java
+@@ -16,6 +16,7 @@ import org.bukkit.event.entity.EntityTransformEvent;
+ import org.bukkit.event.entity.VillagerAcquireTradeEvent;
+ import org.bukkit.event.entity.VillagerReplenishTradeEvent;
+ // CraftBukkit end
++import com.destroystokyo.paper.event.player.PlayerTradeWithVillagerEvent; // Paper
```

Comment:
In NMS, generally one liner imports are omitted.

Reply:
Fixed! Thanks for the feedback.